### ISSUE TITLE
schedule task start 8am erery 5 minutes repeat everyday 

### ISSUE THREAD

**Issue by uct8086** [node-schedule/node-schedule#482]
our PM want a schedule task set time butween 08 am and 10 pm , at the same time , the task repeat every 5 minutes everyday ，can node-schedule do this ?

**Comment by circa10a** [comment]
@uct8086 Yes, that's just a standard cron entry like so: `*/5 08-22 * * *`

Here's a good site for forecasting cron schedules: https://crontab.guru/#*/5_08-22_*_*_*

**Comment by DavinderPRO** [comment]
@circa10a can you please help me out ? 
   I am trying to scheduled a job every 15 days , right have hard coded days 15, 30. 
Is this following code correct for this purpose 
```

    var j = schedule.scheduleJob({
        date: [15, 30],
        hour: 11,
        minute: 00,
    }, sendoutdatedemailnow);
```

**Comment by circa10a** [comment]
@DavinderPRO That should work.

Alternatively,

`0 0 */14 * *`
